src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<dialects>
+    <dialect type="CASSANDRA">
+
+        <procedure name="ScanRecord">
+            <statement name="scanStmt">
+                SELECT * FROM USERTABLE WHERE YCSB_KEY &gt; ? AND YCSB_KEY &lt; ? ALLOW FILTERING
+            </statement>
+        </procedure>
+        <procedure name="ReadModifyWriteRecord">
+            <statement name="selectStmt">
+                Select * from USERTABLE where YCSB_KEY=? 
+           </statement>
+        </procedure>
+
+        <procedure name="InsertRecord">
+            <statement name="insertStmt">
+                INSERT INTO USERTABLE (YCSB_KEY, FIELD1, FIELD2, FIELD3, FIELD4, FIELD5, FIELD6, FIELD7, FIELD8, FIELD9, FIELD10) VALUES (?,?,?,?,?,?,?,?,?,?,?)

```

Comment:
Do you really need this?

Reply:
I think we need this because the original command did not include column names